requirements-test.txt

```diff
@@ -27,4 +27,5 @@ hydra-core >= 1.1
 fairscale >= 0.4.5
 
 # Dependency for fused layers, optional
-triton == 2.0.0.dev20220701
+triton == 2.0.0.dev20221105
```

Comment:
We will have to get @blefaudeux PR with Triton changes in before merging this

Reply:
should be finally really to merge, sorry for the delay I've been a little underwater with some projects..